### ISSUE TITLE
Exception while starting up module "org.imdc.nodered.IgnitionNodeRED".

### ISSUE THREAD

**Issue by apbrauer73** [IgnitionModuleDevelopmentCommunity/IgnitionNode-RED#33]
I imported the node red module v1.5.12 into my Ignition instance Maker Edition v8.1.25 and this is a in a docker container.  The module fails to run.  This also happens with v1.5.11 in Ignition Maker Edition v8.1.24.  The last time this worked was in Maker Edition v8.1.23 with node red module v1.5.11.  I see the following error in the log.

java.lang.IllegalArgumentException: Servlet map already contains "node-red"
at com.inductiveautomation.ignition.gateway.bootstrap.MapServlet.addServlet(MapServlet.java:107)
at com.inductiveautomation.ignition.gateway.web.WebResourceManagerImpl.addServlet(WebResourceManagerImpl.java:241)
at org.imdc.nodered.GatewayHook.startup(GatewayHook.java:38)
at com.inductiveautomation.ignition.gateway.modules.ModuleManagerImpl$LoadedModule.startup(ModuleManagerImpl.java:2439)
at com.inductiveautomation.ignition.gateway.modules.ModuleManagerImpl.startupModule(ModuleManagerImpl.java:1232)
at com.inductiveautomation.ignition.gateway.modules.ModuleManagerImpl$2.call(ModuleManagerImpl.java:777)
at com.inductiveautomation.ignition.gateway.modules.ModuleManagerImpl.executeModuleOperation(ModuleManagerImpl.java:953)
at com.inductiveautomation.ignition.gateway.modules.ModuleManagerImpl.installModuleInternal(ModuleManagerImpl.java:743)
at com.inductiveautomation.ignition.gateway.modules.ModuleManagerImpl$InstallCommand.execute(ModuleManagerImpl.java:1909)
at com.inductiveautomation.ignition.gateway.modules.ModuleManagerImpl$Receiver.receiveCall(ModuleManagerImpl.java:1862)
at com.inductiveautomation.ignition.gateway.redundancy.QueueableMessageReceiver.receiveCall(QueueableMessageReceiver.java:47)
at com.inductiveautomation.ignition.gateway.redundancy.RedundancyManagerImpl.dispatchMessage(RedundancyManagerImpl.java:1030)
at com.inductiveautomation.ignition.gateway.redundancy.RedundancyManagerImpl$ExecuteTask.run(RedundancyManagerImpl.java:1098)
at com.inductiveautomation.ignition.common.execution.impl.BasicExecutionEngine$ThrowableCatchingRunnable.run(BasicExecutionEngine.java:544)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.base/java.lang.Thread.run(Unknown Source)

This is what I see on the module page...
![image](https://user-images.githubusercontent.com/86741406/219875622-294895da-4c2a-4f00-82b3-753a599152bc.png)

**Comment by apbrauer73** [comment]
I figured it out.  I created a new container for Ignition v8.1.25 and I restored a gateway backup from a gateway that was using the node red module.  After the restore I imported the node red module and the exception occurs.  

But if I create the a new Ignition v8.1.25 container and import the node red module first then restore the gateway backup.  The node red module starts up fine in the gateway.  So having the node red module installed before restoring a gateway backup that references the node red module is the way to go.  👍
